scripts/install_from_src_macos

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+
+clean_slate()
+{
+	echo "                      =============================================="
+	echo "                      If you are within a conda env please deactivate it and come back"
+	echo "                      If you don't want to proceed kill this script in the next 10 seconds!"
+	echo "                      =============================================="
+
+	sleep 10
+
+	echo "Cleaning up state"
+	# uninstall any global installations
+	pip uninstall --yes torchserve
+	pip uninstall --yes torch-model-archiver
+	sleep 5
+}
+
+install_java_deps()
+{
+	brew tap AdoptOpenJDK/openjdk
+	brew cask install adoptopenjdk11
+}
+
+install_torch_deps()
+{
+	# replace this with requirements.txt when ready
+	pip install torch torchtext torchvision sentencepiece psutil future
+}
+
+build_install_server()
+{
+	cd ../ && pip install .
+}
+
+build_install_archiver()
+{
+	cd ../model-archiver && pip install .
```

Comment:
since we are running this script from serve_home this should be just `cd model-archiver && pip install .`